src/components/nrc-content/nrc-content-component.jsx

```diff
@@ -137,7 +143,13 @@ function NrcContent({
         </IndicatorCard>
         <IndicatorCard
           color="#008604"
-          indicator={`${Math.round(prop_protected_ter)}%` || ''}
+          indicator={prop_protected_ter && `${Math.round(prop_protected_ter)}%`}
+          description={
+            <p>
+              {t('of')} <b>{t('land is protected')}</b> {t('and')}{' '}
```

Comment:
We need to join this all together as one sentence. Let me know if you need examples